test/evergreen.yml

```diff
@@ -2458,6 +2459,40 @@ tasks:
           upload_source_dir: "wiredtiger"
           upload_filename: unit-test-disagg-follower-tsan.tgz
 
+  - name: format-stress-test-tsan
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test"
+        vars:
+          # Limit the number of operations to avoid an internal TSAN issue; run more operations with detect_deadlocks=0.
+          # FIXME-WT-16313: Consider how to avoid the internal TSAN issue.
+          test_format_extra_args: runs.rows=1000:3000 runs.tables=1:3 runs.ops=3000
+          times: 3
+      - func: "format test"
+        vars:
+          test_format_extra_args: runs.rows=100000:300000 runs.tables=1:3 runs.ops=300000
+          additional_san_vars: export TSAN_OPTIONS="$TSAN_OPTIONS:detect_deadlocks=0"
+          times: 3
+
+  - name: format-stress-test-disagg-leader-tsan
```

Comment:
We should also run follower and switch runs as they could be touching different parts of WT codebase.